components/brave_ads/core/internal/user_engagement/site_visit/site_visit.cc

```diff
@@ -184,11 +184,7 @@ base::TimeDelta SiteVisit::CalculateRemainingTimeToLandOnPage(
 
   const PageLandInfo& page_land = page_lands_[tab_id];
 
-  const base::TimeDelta remaining_time =
-      page_land.timer.desired_run_time() - base::TimeTicks::Now();
-  CHECK(!remaining_time.is_negative());
```

Comment:
Remove this check, because when the page land is resumed negative timers are instantly fired.